docs/en/interfaces/formats.md

```diff
@@ -1309,4 +1310,27 @@ Limitations:
 - In case of parsing error `JSONEachRow` skips all data until the new line (or EOF), so rows must be delimited by `\n` to count errors correctly.
 - `Template` and `CustomSeparated` use delimiter after the last column and delimiter between rows to find the beginning of next row, so skipping errors works only if at least one of them is not empty.
 
+## LineAsString {#lineasstring}
+
+This format parses a sequence of line objects separated by newlines, spaces and/or comma and interprets every line of input data as a single value. This format can only parse a table with a single field of type [String](../sql-reference/data-types/string.md). The remaining columns must be set to [DEFAULT](../sql-reference/statements/create/table.md#default) or [MATERIALIZED](../sql-reference/statements/create/table.md#materialized), or omitted.
```

Comment:
This format parses a sequence of line objects separated by newlines and interprets every line of input data as a single value. This format can only parse a table with a single field of type [String](../sql-reference/data-types/string.md). The remaining columns must be set to [DEFAULT](../sql-reference/statements/create/table.md#default) or [MATERIALIZED](../sql-reference/statements/create/table.md#materialized), or omitted.